features/support/tag_helpers.rb

```diff
@@ -1,3 +1,26 @@
+# Creates a tag with the given name in the local repo
+def create_local_tag tag_name
+  run "git tag -a #{tag_name} -m '#{tag_name}'"
+end
+
+
+# Creates a tag with the given name in the remote repo
+def create_remote_tag tag_name
+  in_repository :origin do
+    create_local_tag tag_name
+  end
+end
+
+
+# Returns a Mortadella instance with all local and remote tags
+def current_tags
```

Comment:
I think table should appear in this method name since this is returning a Mortadella instance